handbook/security.md

```diff
@@ -255,14 +255,96 @@ We configure Chrome on company-owned devices with a basic policy.
+| Allow users with read access to create discussions                         | 🚫                   | We do not currently use discussions, and want people to use issues as much as possible.                                                                                       |
+| Allow members to create teams                                              | 🚫                   | We want to automate the management of GitHub teams with the [GitHub Terraform provider](https://github.com/integrations/terraform-provider-github)                            |
+
+### Team Discussions
+We do not use team discussions, and therefore have disabled it. This is simply to avoid discussions
+being located into to many places, and not security related.
+
+### Repository Security
+
+#### Branch protection
+Branch protection is one of the most important settings to configure, and the main reason we should not have members with administrative privileges on the repositories.
+
+Located in the Branches section of repository settings, we create a rule for **main** that applies:
+
+| Setting                                                          | Value | Note                                                                                                                  |
+| ---------------------------------------------------------------- | ----- | --------------------------------------------------------------------------------------------------------------------- |
+| Require a pull request before merging                            | ✅     | We enforce code reviews, which require PRs.                                                                           |
+| Require approvals                                                | 1️⃣   | We require approval from one person in the team.                                                                      |
+| Dismiss stale pull request approvals when new commits are pushed | ✅     | Without this, someone could get approval for a small, very nice PR then change everything about it!                   |
+| Require review from Code Owners                                  | 🗓     | We are working towards enabling this, but at our current size, it is not needed.                                      |
+| Restrict who can dismiss pull request reviews                    | 🚫     | As we are a team working in multiple timezones, we want to allow dismissing reviews and getting another one.          |
+| Allow specified actors to bypass required pull requests          | 🚫     | We do not want anyone pushing directly to main.                                                                       |
+| Require status checks to pass before merging                     | ✅     | Because of our monorepo, it is hard to pick many checks that work for all types of PRs, but we still enable this.     |
```

Comment:
I'm not familiar with "monorepo." Should we capitalize and explain it?

Reply:
Made it a link instead, since a lot of people might not have heard the term before!